source/loop/apis.rst

```diff
@@ -865,6 +865,18 @@ POST /rooms
     Optional parameter:
 
     - **expiresIn**, the number of hours for which the room will exist.
+    - **channel**, the TokBox channel to use for the call
```

Comment:
That isn't really the tokbox channel, but more the Firefox channel (the channel of the client). We can also explicit here why we need it and what's the actual purpose. Something along the lines of:

`The client can send its channel in order to let the server use different API keys (of the underlying service provider) depending on it`.